.golangci.yml

```diff
@@ -8,21 +11,37 @@ linters:
     - govet
     - errcheck
     - staticcheck
-    - gofmt
-    - goimports
     - revive
     - ineffassign
-    - typecheck
     - unused
-    - gosimple
     - misspell
     - nakedret
     - bodyclose
     - gocritic
     - makezero
     - gosec
+  settings:
+    staticcheck:
+      checks:
+        - all
+        - '-QF1008' # Allow embedded structs to be referenced by field
+        - '-ST1000' # Do not require package comments
+    revive:
+      rules:
+        - name: exported
+          disabled: true
+        - name: exported
+          disabled: true
+        - name: package-comments
+          disabled: true
+
+formatters:
+  enable:
+    - gofmt
```

Comment:
@radar07 I removed `gci` and `golines`. I like them but they aren't included in most people's workflows and the errors can be opaque for example:

```
internal/ghmcp/server.go:17:1: File is not properly formatted (gci)

^
```

Totally unclear how to fix unless you understand what `gci` is. Not worth the hassle.